src/main/resources/templates/fragments/admin-logs.html

```
@@ -1,27 +1,29 @@
-<div th:fragment="content">
-    <h2>Systemlogg</h2>
+<section layout:fragment="content" xmlns:layout="">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)